files/image_config/caclmgrd/caclmgrd

```diff
@@ -247,6 +304,63 @@ class ControlPlaneAclManager(object):
 
                             iptables_cmds.append(rule_cmd)
 
+        # For any services which we did not find ACL configuration for, we add an
+        # iptables rule to allow all incoming packets. This way we don't block
+        # services like SSH on an unconfigured device
+        for acl_service in unconfigured_services:
+            log_info("Did not find or could not apply ACL configuration for {0}. Allowing all incoming traffic for this service...".format(acl_service))
+            ip_protocols = self.ACL_SERVICES[acl_service]["ip_protocols"]
+            dst_ports = self.ACL_SERVICES[acl_service]["dst_ports"]
+            for ip_protocol in ip_protocols:
+                for dst_port in dst_ports:
+                    iptables_cmds.append("iptables -A INPUT -p {0} --dport {1} -j ACCEPT".format(ip_protocol, dst_port))
+                    iptables_cmds.append("ip6tables -A INPUT -p {0} --dport {1} -j ACCEPT".format(ip_protocol, dst_port))
+
+        # Add iptables rules to drop all packets destined for loopback interface IP addresses
```

Comment:
suggest to have a function.

block_ip2me_traffic()

In this function, we should block all traffic destined to my router IP.

This include loopback, manangement, vlan, and p2p IP.

Reply:
Moved "ip2me" command creation into new `generate_block_ip2me_traffic_iptables_commands()` function in latest commit.